### PR TITLE
Attempt to fix 'System Exit has timed out in the forked process 1'

### PR DESCRIPTION
**Pull request by reta**
Attempt to fix 'System Exit has timed out in the forked process 1'

### CODE REVIEW

**Comment by reta** [comment]
@adriancole looks like it helps:

```
18:10:25.108 [main] INFO  o.o.p.e.r.c.RemoteBundleContextClient - Waiting for remote bundle context.. on 21000 name: c6439e46-fd20-49fe-8c0f-ed9b87bed048 timout: [ RelativeTimeout value = 180000 ]

WARNING: An illegal reflective access operation has occurred

WARNING: Illegal reflective access by org.apache.felix.framework.ext.ClassPathExtenderFactory$DefaultClassLoaderExtender (file:/home/travis/.pax/exam/system/org/apache/felix/org.apache.felix.framework/5.6.12/org.apache.felix.framework-5.6.12.jar) to method java.net.URLClassLoader.addURL(java.net.URL)

WARNING: Please consider reporting this to the maintainers of org.apache.felix.framework.ext.ClassPathExtenderFactory$DefaultClassLoaderExtender

WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations

WARNING: All illegal access operations will be denied in a future release

18:10:40.432 [main] INFO  o.o.p.e.spi.reactors.ReactorManager - suite finished

[INFO] Tests run: 8, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 17.281 s - in io.zipkin.brave.itests.ITBrave

[INFO] Tests run: 8, Failures: 0, Errors: 0, Skipped: 0

[INFO] 

[INFO] --- maven-failsafe-plugin:3.0.0-M5:integration-test (integration-test) @ brave-itests ---

[INFO] Skipping execution of surefire because it has already been run for this configuration
```

On a side note, I have seen A LOT of issues popping up with failsafe / surefire M4/M5, this seems to be one of those :(

File: itests/pom.xml [hunk 1]
```diff
@@ -167,6 +167,9 @@
       </plugin>
       <plugin>
         <artifactId>maven-failsafe-plugin</artifactId>
+        <configuration>
+            <forkNode implementation="org.apache.maven.plugin.surefire.extensions.SurefireForkNodeFactory"/>
```
Comment (codefromthecrypt) [inline itests/pom.xml:172]:
```suggestion
            <!-- Prevents 'System Exit has timed out in the forked process 1' -->
            <forkNode implementation="org.apache.maven.plugin.surefire.extensions.SurefireForkNodeFactory"/>
```

**Comment by codefromthecrypt** [comment]
Thanks for looking into this @reta. Indeed there have been some glitches between new plugins and the lightning speed JDKs

**Comment by reta** [comment]
Thanks @adriancole ! Should be good now :)

**Comment by codefromthecrypt** [comment]
awesome and thanks!

**Pull request merged**